docs/assets/templates/tools/tools.rst

```diff
@@ -0,0 +1,28 @@
+.dockstore.yml for Tools Templates (version 1.2)
+================================================
+Several templates and examples are provided here for you to reference for your own .dockstore.yml files. The last example provides a complete explanation of the possible fields and values you can use.
+
+Simple generic template for a tool
+----------------------------------
+.. include:: /assets/templates/tools/barebones-.dockstore.yml
```

Comment:
Hmmm, barebones really is `simple` and barebones. I would strip out the optional fields then and keep them for the `filled-out` or `full` templates below